src/cmap/commands.ts

```diff
@@ -75,10 +74,13 @@ export class Query {
   partial: boolean;
   documentsReturnedIn?: string;
 
-  constructor(ns: string, query: Document, options: OpQueryOptions) {
+  constructor(public databaseName: string, query: Document, options: OpQueryOptions) {
```

Comment:
for both constructors in this file - can we either declare all class properties in the constructor using `public/private` syntax or none?  whatever we choose, let's be consistent

Reply:
I agree that will be easier for readability and consistency reasons.